pkg/platform/api/buildplanner/model/buildplan.go

```diff
@@ -343,6 +343,11 @@ func ProcessCommitError(commit *Commit, fallbackMessage string) error {
 			commit.Type, commit.Message,
 			locale.NewInputError("err_buildplanner_parse_error", "The platform failed to parse the build expression, received message: {{.V0}}. Path: {{.V1}}", commit.Message, commit.ParseError.Path),
 		}
+	case ValidationErrorType:
```

Comment:
This isn't directly related to this story but I found that we weren't handling this error type so I've added it here.